index.js

```diff
@@ -0,0 +1,186 @@
+const popup = document.querySelector('.overlay');
+const closePopup = document.querySelector('#closebtn');
+const subDesc = document.querySelector('#popsubdesc');
+const workImg = document.querySelector('#popMimg');
+const deskImg = document.querySelector('#popDimg');
+const popupTitle = document.querySelector('#poptitle');
+const popupContent = document.querySelector('.pop-work');
+const skillsDiv = document.querySelector('#poplangs');
+const workDesc = document.querySelector('#popdesc');
+const liveBtn = document.querySelector('#livebtn');
+const sourceBtn = document.querySelector('#codebtn');
+const Portfolio = document.querySelector('#Portfolio');
+
+function poping() {
+  if (pop.style.display === 'block') {
+    menu.src = 'img/icon-hamburger.png';
+    pop.style.display = 'none';
+  } else {
+    menu.src = 'img/close-button.png';
+    pop.style.display = 'block';
+  }
+}
+
+menu.addEventListener('click', pop);
+
+const menus = document.querySelectorAll('.pop-menu a');
+for (let i = 0; i < menus.length; i += 1) {
+  menus[i].addEventListener('click', () => {
+    poping();
+  });
+}
+
+// popup generator
+
+const project = [{
+  name: 'Tonic',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  Image: 'img/desktop-project-1.png',
+  technologies: [
+    'HTML',
+    'CSS',
+    'JavaScript',
+  ],
+  liveVersionLink: '',
+  SourceLink: '',
+  company: 'CANOPY',
+  year: '2015',
+  stack: 'Back End Dev',
+},
+{
+  name: 'Post Stories',
+  description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+  Image: 'img/desktop-project-2.png',
+  technologies: [
+    'HTML',
+    'Ruby on Rails',
+    'CSS',
+    'JavaScript',
+  ],
+  liveVersionLink: '',
+  SourceLink: '',
+  company: 'FACEBOOK',
+  year: '2015',
+  stack: 'Full Stack Dev',
+},
+{
+  name: 'Facebook 360',
+  description: 'Exploring the future of media in Facebook\'s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+  Image: 'img/desktop-project-3.png',
+  technologies: [
+    'HTML',
+    'Ruby on Rails',
+    'CSS',
+    'JavaScript',
+  ],
+  liveVersionLink: '',
+  SourceLink: '',
+  company: 'FACEBOOK',
+  year: '2015',
+  stack: 'Full Stack Dev',
+},
+{
+  name: 'Uber Navigation',
+  description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+  Image: 'img/desktop-project-4.png',
+  technologies: [
+    'HTML',
+    'Ruby on Rails',
+    'CSS',
+    'JavaScript',
+  ],
+  liveVersionLink: '',
+  SourceLink: '',
+  company: 'Uber',
+  year: '2018',
+  stack: 'Lead Developer',
+}];
+
+const generatePopup = (index) => {
+  skillsDiv.innerHTML = '';
+  popupTitle.innerText = project[index].name;
+  workImg.src = project[index].mobileImage;
+  deskImg.src = project[index].desktopImage;
```

Comment:
- Good job done so far, I currently cannot see the project image in the popup, a likely reason for this is that you are trying to access the `mobileImage` key in the project array, which is not present. Please fix this 👍 👍. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
- I suggest you consider using the same image for the selected project in that projects array 😄 
![ehsethse](https://github.com/obibaadoma/potfolio-desktop/assets/73607512/2f45eec8-215c-451c-9c7e-16b5f1a4ee8f)

Reply:
added image links